.github/workflows/schema-compose.yaml

```diff
@@ -0,0 +1,93 @@
+name: Schema
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  generate:
+    # Deduplicate jobs from pull requests and branch pushes within the same repo.
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4.1.4
+
+      - name: Install Rover CLI
+        run: |
+          curl -sSL https://rover.apollo.dev/nix/v0.23.0-rc.3 | sh
```

Comment:
How are we going to keep this updated?

Reply:
Manually I guess? I found a github action in market place but its using older version I guess. Any way we only using it to compose supergraphs, should have any issues unless there's a major updates?